src/main/java/reposense/RepoSense.java

```diff
@@ -126,8 +126,11 @@ public static void main(String[] args) {
 
     /**
      * Constructs a list of {@code RepoConfiguration} if {@code cliArguments} is a {@code LocationsCliArguments}.
+     *
+     * @throws InvalidLocationException if all repo locations are invalid.
```

Comment:
It is not so ideal to use `InvalidLoactionException`. However, it is still acceptable.

Reply:
Thank you for your suggestion! So should I implement another exception class to handle this?

Reply:
I think `ParseException` will be suitable?

Reply:
sure

Reply:
@shuiyao-sg, could you update the exception? Thanks!

Reply:
sure, I'll update it now.